code/modules/mob/dead/new_player/new_player.dm

```diff
@@ -431,46 +431,81 @@
-		for(var/datum/job/a in SSjob.prioritized_jobs)
-			amt_count++
-			if(amt_count != amt) // checks for the last job added.
-				dat += " [a.title], "
-			else
-				dat += " [a.title]. </div>"
+	if(!available_job_count)
+		dat += "<div class='notice red'>There are currently no open positions!</div>"
 
-	dat += "<div class='clearBoth'>Choose from the following open positions:</div><br>"
-	dat += "<div class='jobs'><div class='jobsColumn'>"
-	var/job_count = 0
-	for(var/datum/job/job in SSjob.occupations)
-		if(job && IsJobUnavailable(job.title, TRUE) == JOB_AVAILABLE)
-			job_count++;
-			if (job_count > round(available_job_count / 2))
-				dat += "</div><div class='jobsColumn'>"
-			var/position_class = "otherPosition"
-			if (job.title in GLOB.command_positions)
-				position_class = "commandPosition"
-			dat += "<a class='[position_class]' href='byond://?src=[REF(src)];SelectedJob=[job.title]'>[job.title] ([job.current_positions])</a><br>"
-	if(!job_count) //if there's nowhere to go, overflow opens up.
+	else
+
+	// if(length(SSjob.prioritized_jobs))
+	// 	dat += "<div class='notice red'>The station has flagged these jobs as high priority:<br>"
+	// 	for(var/datum/job/a in SSjob.prioritized_jobs)
+	// 		dat += " [a.title], "
+	// 	dat += "</div>"
+
+		dat += "<div class='clearBoth'>Choose from the following open positions:</div><br>"
+		var/list/categorizedJobs = list(
+			"Command" = list(jobs = list(), titles = GLOB.command_positions, color = "#aac1ee"),
+			"Engineering" = list(jobs = list(), titles = GLOB.engineering_positions, color = "#ffd699"),
+			"Supply" = list(jobs = list(), titles = GLOB.supply_positions, color = "#ead4ae"),
+			"Miscellaneous" = list(jobs = list(), titles = list(), color = "#ffffff", colBreak = 1),
+			"Synthetic" = list(jobs = list(), titles = GLOB.nonhuman_positions, color = "#ccffcc"),
+			"Service" = list(jobs = list(), titles = GLOB.civilian_positions, color = "#cccccc"),
+			"Medical" = list(jobs = list(), titles = GLOB.medical_positions, color = "#99ffe6", colBreak = 1),
+			"Science" = list(jobs = list(), titles = GLOB.science_positions, color = "#e6b3e6"),
+			"Security" = list(jobs = list(), titles = GLOB.security_positions, color = "#ff9999"),
+		)
 		for(var/datum/job/job in SSjob.occupations)
-			if(job.title != SSjob.overflow_role)
+			if(job && IsJobUnavailable(job.title, TRUE) == JOB_AVAILABLE)
+				var/categorized = 0
+				for(var/jobcat in categorizedJobs)
+					var/list/jobs = categorizedJobs[jobcat]["jobs"]
+					if(job.title in categorizedJobs[jobcat]["titles"])
+						categorized = 1
+						if(jobcat == "Command")
+
+							if(job.title == "Captain") // Put captain at top of command jobs
+								jobs.Insert(1, job)
+							else
+								jobs += job
+						else // Put heads at top of non-command jobs
+							if(job.title in GLOB.command_positions)
+								jobs.Insert(1, job)
+							else
+								jobs += job
+				if(!categorized)
+					categorizedJobs["Miscellaneous"]["jobs"] += job

```

Comment:
![image](https://user-images.githubusercontent.com/20558591/58579461-6a5abb80-824b-11e9-90c2-b7de0c3a7f52.png)

Reply:
Isnt this where Assistant lies?

Reply:
Wait its under service